vocabularies/Org.OData.Aggregation.V1.xml

```diff
@@ -76,7 +76,17 @@
       </Term>
       <Term Name="ApplySupportedDefaults" Type="Aggregation.ApplySupportedBase" Nullable="false" AppliesTo="EntityContainer">
         <Annotation Term="Core.Description">
-          <String>Default support of the `$apply` system query option for all collection-valued resources in the container. Annotating term `ApplySupported` for a specific collection-valued resource overrides the default setting</String>

```

Comment:
"default setting" could be more precisely specified as "annotation of term Aggregation.ApplySupportedDefaults" to make it easier to understand

Reply:
Changed long description below to use "default support" instead of "default setting" - "defaullt support" is used in the short description.